doc/policies/admin.rst

```diff
@@ -470,4 +470,15 @@ type: bool
 
 The administrator is allowed to export a complete system documentation
 including resolvers and realm.
-The documentation is created as restructured text.
\ No newline at end of file
+The documentation is created as restructured text.
+
+sms_gateways
+~~~~~~~~~~~~
+
+type: string
+
+Usually an SMS tokens sends the SMS via the SMS gateway that is system wide defined in the token settings.
```

Comment:
... 'a' SMS token_ ... which is defined system wide in ...

Reply:
would like to not fix.